examples/conda/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM continuumio/miniconda3:4.10.3
+
+WORKDIR /test
+RUN conda install -y conda-build
+RUN conda install -y conda-verify
+RUN conda install -y anaconda-client
+RUN anaconda config --set url "http://artipie:8080/my-conda/" -s
+RUN conda config --set anaconda_upload yes
+RUN echo "channels:\r\n  - http://artipie:8080/my-conda" > /root/.condarc
```

And here (as fewer `RUN` command we have, as faster Docker builds images; each `RUN` produces image layer which is not really fast operation, see https://docs.docker.com/develop/develop-images/dockerfile_best-practices/):
 ```suggestion
RUN anaconda config --set url "http://artipie:8080/my-conda/" -s && \
    conda config --set anaconda_upload yes && \
    echo "channels:\r\n  - http://artipie:8080/my-conda" > /root/.condarc
```